src/main/scala/is/hail/utils/richUtils/RichInt.scala

```diff
@@ -4,4 +4,6 @@ import is.hail.utils.UInt
 
 class RichInt(val i: Int) extends AnyVal {
   def toUInt: UInt = UInt(i)
+
+  def toUIntNoCheck: UInt = UInt.applyUnchecked(i)
```

Comment:
I like names that say what (convert a UInt representation stored in an Int to a UInt) rather that how (by not checking i >= 0).